restcomm/restcomm.ussd/src/main/java/org/restcomm/connect/ussd/interpreter/UssdInterpreter.java

```diff
@@ -490,7 +490,17 @@ public void onReceive(final Object message) throws Exception {
                 @SuppressWarnings("unchecked")
                 final CallResponse<CallInfo> response = (CallResponse<CallInfo>) message;
                 // Check from whom is the message (initial call or outbound call) and update info accordingly
-                if (sender == ussdCall) {
+
```

Comment:
@maria-farooq 

I am fine with that, please go ahead if you feel that it is a better option.


Charles